sandbox/react-app/main.tsx

```diff
@@ -1,18 +1,32 @@
-import { Link, Outlet, RouterProvider, useParams } from 'react-router-dom'
-import React, { useEffect, useState } from 'react'
-import ReactDOM from 'react-dom/client'
+import { createDatadogProvider } from '@datadog/browser-flagging'
 import { datadogRum } from '@datadog/browser-rum'
+import { ErrorBoundary, reactPlugin, UNSTABLE_ReactComponentTracker } from '@datadog/browser-rum-react'
 import { createBrowserRouter } from '@datadog/browser-rum-react/react-router-v7'
-import { reactPlugin, ErrorBoundary, UNSTABLE_ReactComponentTracker } from '@datadog/browser-rum-react'
-import { datadogFlagging } from '@datadog/browser-flagging'
+import { OpenFeature } from '@openfeature/web-sdk'
+import React, { useEffect, useState } from 'react'
+import ReactDOM from 'react-dom/client'
+import { Link, Outlet, RouterProvider, useParams } from 'react-router-dom'
 
 datadogRum.init({
   applicationId: 'xxx',
   clientToken: 'xxx',
   plugins: [reactPlugin({ router: true })],
 })
 
-datadogFlagging.init()
+const subject = {
+  key: 'subject-key-1',
+}
+
+async function initializeOpenFeature() {
+  const datadogFlaggingProvider = createDatadogProvider()
+  await OpenFeature.setContext(subject)
```

Comment:
presumably after this is where we could also call `setLogger()` ([package docs](https://www.npmjs.com/package/@openfeature/web-sdk#logging))